apps/apps.c

```diff
@@ -2294,29 +2294,28 @@ int app_access(const char* name, int flag)
 #ifdef _WIN32
 int app_isdir(const char *name)
 {
-    HANDLE hList;
-    WIN32_FIND_DATA FileData;
+    DWORD attr;
 # if defined(UNICODE) || defined(_UNICODE)
     size_t i, len_0 = strlen(name) + 1;
+    WCHAR tempname[MAX_PATH];
 
-    if (len_0 > OSSL_NELEM(FileData.cFileName))
+    if (len_0 > MAX_PATH)
         return -1;
 
 #  if !defined(_WIN32_WCE) || _WIN32_WCE>=101
     if (!MultiByteToWideChar
-        (CP_ACP, 0, name, len_0, FileData.cFileName, len_0))
+        (CP_ACP, 0, name, len_0, tempname, len_0))
```

Comment:
> Or do you want me to change the second len_0 to MAX_PATH?

"Side note" means that no action is expected. But it doesn't mean that you can't ponder over it. You might get a viable idea. And suggestion to replace len_0 with MAX_PATH can be viewed as readability improvement... Your call :-)